src/callout.tsx

```diff
@@ -107,7 +111,11 @@ const Callout: React.FC<Props> = ({
                                 bleedRight
                                 Icon={IconCloseRegular}
                                 onPress={onClose}
-                                aria-label={closeButtonLabel ?? texts.closeButtonLabel}
+                                aria-label={
+                                    closeButtonLabel ||
+                                    texts.closeButtonLabel ||
+                                    translate(closeButtonLabelText, locale)
```

Comment:
You will see this pattern a lot in following files:

```js
texts.patata || translate(patata)
```
I'm unable to think way to avoid having to write this in each token usage (unless we disable text customization)

Reply:
Maybe we could have some kind of utility function that does this?

const getTranslation = (token) => texts[token] || translate(token, locale)

Reply:
that wouldn't work.

Imagine you want to translate the text "cancel"

```
const getTranslation = (token) => { // ----> is this token value the "cancel" string?
    if (texts.token) { // --->  this would be `texts['cancel']`
        return token;
    }
    translate(???, locale);  // ---> the translate function accepts as parameter a `Token` type, which is an object
    // like {es: 'cancelar', en: 'cancel'...}. We can't use the "cancel" string here as parameter
}

Reply:
Hmm, I see :(

I think we can proceed with what we have now, and if we come up with some ideas in the future, we can iterate on this